x-pack/plugins/ml/common/types/anomaly_detection_jobs/summary_job.ts

```diff
@@ -14,32 +14,140 @@ import type { MlJobBlocked } from './job';
 export type { Datafeed } from './datafeed';
 export type { DatafeedStats } from './datafeed_stats';
 
+/**
+ * A summary of an anomaly detection job.
+ */
 export interface MlSummaryJob {
+  /**
+   * The identifier for the anomaly detection job.
+   */
   id: string;
+
+  /**
+   * A description of the job.
+   */
   description: string;
+
+  /**
+   * A list of job groups. A job can belong to no groups or many.
```

Comment:
Nit: Can it belong to just one group too, does “many” include that? What about `A job can belong to no groups, one or many`.

Reply:
Done in https://github.com/elastic/kibana/pull/123813/commits/c7856e2f3dd603acdbb6d49fab601a2c00bfea2b